scss/navbar.scss

```diff
@@ -0,0 +1,27 @@
+@import 'colors';
```

Comment:
import this in `application.scss` so you don't have to do it every time

Reply:
Fixed in PR #11 😬